WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -0,0 +1,119 @@
+import Yosemite
+import SwiftUI
+
+struct CollapsibleProductRowCard: View {
+    @ObservedObject var viewModel: ProductRowViewModel
+    @State private var isCollapsed: Bool = true
+
+    private var isExpanded: Binding<Bool> {
+        Binding<Bool>(
+            get: { !self.isCollapsed },
+            set: { self.isCollapsed = !$0 }
+        )
+    }
```

Comment:
Happy to hear if you have a better suggestion for this bit of logic, ideally I wanted to use `!isCollapsed` when passing the property down to the `CollapsibleView`, however since uses a Binding<Bool> type for 2-way communication I haven't found a better way to do it.

Reply:
The computed property makes it much cleaner, since it failed for me to pass the `!$isCollapsible` binding directly to the initializer, I didn't think about this solution. Thanks! Changed here: 793191caf8d4197bd4418e4ffa94241a9b841ca4

> I wrote the above thinking that you'd added CollapsibleView in this PR; I see it's pre-existing. I still think all this applies, but it does depend where/how much we're already using CollapsibleView elsewhere.

Agree, when revising the design for the collapsible component it might be worth to take this approach